hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -169,25 +171,37 @@ public static synchronized void writeDatanodeDetailsTo(
    * @return {@link DatanodeDetails}
    * @throws IOException If the id file is malformed or other I/O exceptions
    */
-  public static synchronized DatanodeDetails readDatanodeDetailsFrom(File path)
```

Comment:
nit: Can we avoid this non meaningful refactoring here. We can do this as part of refactoring jira